examples/component/http-client/README.md

```diff
@@ -0,0 +1,83 @@
+# TinyGo HTTP Client
+
+This repository contains a WebAssembly Component written in [TinyGo][tinygo], which:
+
+- Implements a [`wasi:http`][wasi-http]-compliant HTTP handler
+- Uses the [`httpserver` provider][httpserver-provider] to serve requests
+- Calls a random number generator service to get random numbers using [`wasi:http`][wasi-http]
+- Can be declaratively provisioned with [`wadm`][wadm]
+
+[wasi-http]: https://github.com/WebAssembly/wasi-http
+[httpserver-provider]: https://github.com/wasmCloud/wasmCloud/tree/main/crates/provider-http-server
+[httpclient-provider]: https://github.com/wasmCloud/wasmCloud/tree/main/crates/provider-http-client
+[wadm]: https://github.com/wasmCloud/wadm
+[tinygo]: https://tinygo.org/getting-started/install/
+[wash]:  https://wasmcloud.com/docs/ecosystem/wash/
+
+# Dependencies
+
+Before starting, ensure that you have the following installed:
+
+- The [TinyGo toolchain][tinygo]
+- [`wash`, the WAsmcloud SHell][wash] installed.
+
+# Quickstart
+
+To get started developing this repository quickly, clone the repo and run `wash dev`:
+
+```console
+wash dev
+```
+
+`wash dev` does many things for you:
+
+- Starts the [wasmCloud host][wasmcloud-host] that can run your WebAssembly component
+- Builds this project
+- Builds a declarative WADM manifest consisting of:
+  - Your locally built component
+  - A [HTTP server provider][httpserver-provider] which will receive requests from the outside world
+    (on port 8000 by default)
+  - A [HTTP client provider][httpclient-provider] which will call a random number generator service
+  - Necessary links between providers and your component so your component can handle web traffic
+- Deploys the built manifest (i.e all dependencies to run this application) locally
+- Watches your code for changes and re-deploys when necessary.
+
+[wasmcloud-host]: https://wasmcloud.com/docs/concepts/hosts
+
+## Send a request to the running component
+
+Once `wash dev` is serving your component, to send a request to the running component (via the HTTP
+server provider). It will call an upstream API and return a list of random numbers:
+
+```console
+curl localhost:8000
+[438,424,166,260,681]
+```
```

Comment:
This [example uses port `30001`](https://github.com/wasmCloud/go/blob/main/examples/component/http-client/wadm.yaml#L43), would you mind either updating the README or the `wadm.yaml` to match?

Reply:
Wash dev defaults it to port 8000, so the instructions are correct. I can update the wadm manifest to match though